test/XRP/fakes/fake-xrp-network-client.ts

```diff
@@ -1,5 +1,4 @@
 /* eslint-disable class-methods-use-this */
```

Comment:
a little bit of updating to our fake network client because it's now being used to test reliable submission logic, and thus needed a little more nuance